apps/taxes/src/modules/taxjar/taxjar-webhook.service.ts

```diff
@@ -61,6 +63,12 @@ export class TaxJarWebhookService implements ProviderWebhookService {
   }
 
   async cancelOrder(payload: OrderCancelledEventSubscriptionFragment) {
-    // TaxJar isn't implemented yet
+    // todo: implement
+    this.logger.debug("cancelOrder not implemented for TaxJar");
+  }
+
+  async refundTransaction(payload: OrderFullyRefundedPayload) {
+    // todo: implement
+    this.logger.debug("refundOrder not implemented for TaxJar");
```

Comment:
Maybe this should be a `warn`? Again we won't see that someone tried to use this operation in TaxJAr in our logs